subsys/modem/Kconfig

```diff
@@ -14,8 +14,8 @@ config MODEM_CHAT
 
 if MODEM_CHAT
 
-config MODEM_CHAT_LOG_BUFFER
-	int "Modem chat log buffer size"
+config MODEM_CHAT_LOG_BUFFER_SIZE
+	int "Modem chat log buffer size in bytes"
```

Comment:
This affects users... Do we need this mentioned in release notes some where?

Reply:
I think migration guide would be the place:
https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/releases/migration-guide-3.7.rst

Reply:
Good catch! We need to add this change to the migration guide :)